### PR TITLE
noob airdrop feature

### PR DESCRIPTION
**Pull request by 0xCorbs**

### CODE REVIEW

File: src/NoobAirdrop.sol [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    constructor(
+        bytes32 _merkleRoot,
+        IERC20 _noobToken,
+        address owner
+    ) Ownable(owner) {
+        merkleRoot = _merkleRoot;
+        noobToken = _noobToken;
+    }
+
+    function claimAirDrop(
```
Comment (0xPilot) [inline src/NoobAirdrop.sol:30]: Buggy contract code.

File: src/NoobAirdrop.sol [hunk 2]
```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {BitMaps} from "@openzeppelin/contracts/utils/structs/BitMaps.sol";
+import {MerkleProof} from "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract NoobAirdrop is Ownable {
```
Comment (0xPilot) [inline src/NoobAirdrop.sol]: The function to withdraw the unclaimed tokens was missed.

File: src/NoobAirdrop.sol [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    using SafeERC20 for IERC20;
+
+    bytes32 public merkleRoot;
+    BitMaps.BitMap private _airdropList;
```
Comment (0xPilot) [inline src/NoobAirdrop.sol]: Optimize contract by using mapping variable directly.

File: src/NoobAirdrop.sol (hunk 1 above)
Comment (0xCorbs) [reply src/NoobAirdrop.sol:26]: Can you elaborate?

Comment (Ziden) [reply src/NoobAirdrop.sol:30]: Why ?

**Pull request merged**